ci/nightly-build.sh

```diff
@@ -26,10 +26,22 @@ MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -B"
 CUDA_VER=${CUDA_VER:-cuda`nvcc --version | sed -n 's/^.*release \([0-9]\+\)\..*$/\1/p'`}
 PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
 USE_GDS=${USE_GDS:-ON}
+USE_SANITIZER=${USE_SANITIZER:-ON}
+BUILD_FAULTINJ=${BUILD_FAULTINJ:-ON}
+ARM64=${ARM64:-false}
+
+profiles="source-javadoc"
+if [ "${ARM64}" == "true" ]; then
```

Comment:
I wonder if we should standardize on 'ON' 'OFF' for all Boolean switches.

Reply:
Currently ON/OFF is for cmake config, 
the added `ARM64` ENV in CI script is for CI building only, it looks more clear (true/false) to me in the pipeline setup.
But if we decide to standardize all switches to follow cmake style, I am fine to update all.